index.js

```diff
@@ -70,6 +73,11 @@ function createServer(argv) {
     var ldp = ldpApp.locals.ldp;
     app.use(ldp.mount, ldpApp);
 
+    //Error handling
+    app.use(errorHandler.handler);
```

Comment:
Shouldn't this be mounted as a route?
Mounting this here, you could miss LinkHandler and Cors.

In fact ldpApp.locals.ldp already has the lpd instance